app/assets/stylesheets/course.css.scss

```diff
@@ -1,7 +1,5 @@
-.list-group-item.course {
-  min-height:425px;
-
-  h3 { margin-top:10px; text-align:center; }
+.course {
+  h1 { margin-top:10px; font-size:22px; }
```

Comment:
Properties should be ordered font-size, margin-top<br>Colon after property should be followed by one space